Sonify/README.md

```diff
@@ -0,0 +1,34 @@
+# Sonify README
+
+## Description
+
+Sonify is a CODAP Plugin that permits time series data in
+CODAP to be played as sounds.
+
+## Developer Notes
+
+### Setup
+
+Assumes node and npm are installed.
+
+    `npm install`
+
+### Build
+
+    `npm run build`
+
+### Running locally
+
+Assumes a local CODAP instance and that the working directory is within the
+filesystem managed by a local web server.
+
+- Open CODAP in one tab of a browser.
+- Open the Sonify plugin in another tab.
+- Drag the URL for the latter tab into the tab of the former.
+  Wait for the CODAP tab to become active, then drop the URL in the CODAP
+  workspace.
+
+Alternatively, if CODAP's URL is http://localhost:8080/CODAP/ and the plugin's
+URL is http://localhost:8080/Sonify/, then, invoke them together this way:
+
+    `http://localhost:8080/CODAP/?di=http://localhost:8080/Sonify/`
```

Comment:
Thank you for the README! 

Also, for testing the plugin with a release version of CODAP on `https`, I did the following:
- Install [mkcert](https://github.com/FiloSottile/mkcert). On Mac: `brew install mkcert`
- Create and install the CA in KeyChain with `mkcert -install`
- `cd` to this directory.
- `mkcert -key-file key.pem -cert-file cert.pem localhost`
- `npx http-server . -S` => Go to `https://localhost:8080/Sonify/`

Reply:
I'll add that.